src/windowsservercore/2004/helix/webassembly/Dockerfile

```
@@ -0,0 +1,35 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-2004-helix-amd64
+
+# Install git
+ENV GIT_VERSION=2.31.1
+ENV GIT_INSTALLER=MinGit-${GIT_VERSION}-64-bit.zip
+
+RUN curl -SL --output %TEMP%\%GIT_INSTALLER% https://github.com/git-for-windows/git/releases/download/v2.31.1.windows.1/%GIT_INSTALLER% \
+    && powershell -Command "Expand-Archive %TEMP%\$env:GIT_INSTALLER -DestinationPath C:\git" \
```

A bit simpler to use tar.

```suggestion
    && tar -C C:\git -zxf %TEMP%\%GIT_INSTALLER% \
```